extra/utils/Permissions.js

```diff
@@ -0,0 +1,17 @@
+import permissions from '../constant';
+export default function hasPermission(moduleName,role,permissionType)
+{
+    if(!moduleName.hasOwnProperty(permissionType)){
+        console.log('false');
+    }
+    else if(moduleName[permissionType].includes(role) || (role=='head-trainer')){
+        console.log('true');
+    }
+    else{
+        console.log('false');
+    }
+}
+// console.log(hasPermission('getUsers','head-trainer','all'));
```

Comment:
create the function as instructed in task, avoid changing the correct things

Reply:
Due to some confusion in reading the task by mistake i create the wrong function.